pkg/datarepair/checker/checker.go

```diff
@@ -140,3 +142,19 @@ func lookupResponsesToNodes(responses *pb.LookupResponses) []*pb.Node {
 	}
 	return nodes
 }
+
+func dataAccounting(healthyNodeIds []string, pointer *pb.Pointer) {
```

Comment:
Create a separate thing that is responsible for this, rather than having plain funcs.

Reply:
Could you clarify what you mean here? Thanks!

Reply:
```
type accounting struct {}

func (accounting *accounting) AddSize(nodes []string, pointer *pb.Pointer) error {
```

Reply:
PS: or similar... there probably are better names for these things.